lampod/src/ln/channel_manager.rs

```diff
@@ -426,13 +426,24 @@ impl ChannelEvents for LampoChannelManager {
                 // FIXME: LDK should return a better error struct here
             )
             .map_err(|err| error::anyhow!("{:?}", err))?;
+
+        // Wait for SendRawTransaction to be received so to get the funding transaction
+        let tx: Option<Transaction> = loop {
+            let events = self.handler().events();
+            let event = events.recv_timeout(std::time::Duration::from_secs(30))?;
+
+            if let Event::OnChain(OnChainEvent::SendRawTransaction(tx)) = event {
+                break Some(tx);
+            }
+        };
```

Comment:
We just need a fixme here that tell us that we can going looping forever, currently a fixme is fine

Reply:
Good catch. Thank you